ui/pages/withdraw.tsx

```diff
@@ -0,0 +1,107 @@
+import { Arbitrum, Sepolia } from '@thirdweb-dev/chains'
+import { useAddress, useContract } from '@thirdweb-dev/react'
+import ERC20 from 'const/abis/ERC20.json'
+import {
+  MOONEY_ADDRESSES,
+  VMOONEY_ADDRESSES,
+  VOTING_ESCROW_DEPOSITOR_ADDRESSES,
+  MOONEY_DECIMALS,
+} from 'const/config'
+import { BigNumber } from 'ethers'
+import { useRouter } from 'next/router'
+import React, { useContext } from 'react'
+import toast from 'react-hot-toast'
+import toastStyle from '../lib/marketplace/marketplace-utils/toastConfig'
+import useWindowSize from '@/lib/team/use-window-size'
+import ChainContext from '@/lib/thirdweb/chain-context'
+import useWithdrawAmount from '@/lib/utils/hooks/useWithdrawAmount'
+import Container from '../components/layout/Container'
+import ContentLayout from '../components/layout/ContentLayout'
+import WebsiteHead from '../components/layout/Head'
+import StandardButton from '../components/layout/StandardButton'
+import Asset from '@/components/dashboard/treasury/balance/Asset'
+import { NoticeFooter } from '@/components/layout/NoticeFooter'
+import NetworkSelector from '@/components/thirdweb/NetworkSelector'
+
+export default function Withdraw() {
+  const userAddress = useAddress()
+  const router = useRouter()
+  const chain = process.env.NEXT_PUBLIC_CHAIN === 'mainnet' ? Arbitrum : Sepolia
+  const { contract: votingEscrowDepositorContract } = useContract(
+    VOTING_ESCROW_DEPOSITOR_ADDRESSES[chain.slug]
```

Comment:
pretty sure adding the abi here speeds things up so thirdweb doesn't need to fetch it